dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java

```diff
@@ -3356,17 +3366,14 @@ private DynoJedisClient buildDynoJedisClient() {
 
             setLoadBalancingStrategy(cpConfig);
 
-            DynoCPMonitor cpMonitor = new DynoCPMonitor(appName);
-
-
             JedisConnectionFactory connFactory = new JedisConnectionFactory(opMonitor);
 
             return startConnectionPool(appName, connFactory, cpConfig, cpMonitor);
         }
 
         private ConnectionPoolImpl<Jedis> startConnectionPool(String appName, JedisConnectionFactory connFactory,
                                                               ConnectionPoolConfigurationImpl cpConfig,
-                                                              DynoCPMonitor cpMonitor) {
+                                                              ConnectionPoolMonitor cpMonitor) {
```

Comment:
Changed the argument type to ConnectionPoolMonitor so the customer implementations can be accepted